src/components/accounts/EnterVerificationCode.js

```diff
@@ -76,17 +71,18 @@ const SetRecoveryMethodSuccess = ({
 
     return (
         <Container className='ui container' onSubmit={e => {onConfirm(code); e.preventDefault();}}>
-            <h1><Translate id='setRecoveryConfirm.pageTitle'/> {useEmail ? 'Email' : 'Phone'}</h1>
-            <div className='desc one'><Translate id='setRecoveryConfirm.pageText'/> {useEmail ? email : phoneNumber}</div>
+            <h1>Enter Verification Code</h1>
```

Comment:
Let's not make strings non-translatable

Reply:
@vgrichina there's a lot of work left here I'm just adding base UI